app/pages/profile/page.tsx

```diff
@@ -1,11 +1,16 @@
+import LogoutButton from "@/components/LogoutButton";
+
 export default function Profile() {
-    return (
-        <div>
-            <div className="flex flex-col justify-center items-center text-3xl">
-                <div className="font-bold w-1/5 text-center py-[200px]">
-                    Welcome to your profile page.
-                </div>
-            </div>
+  return (
+    <div>
```

Comment:
Surely we can remove most of these `div`s?

Reply:
I have removed the top-level div